### DISCUSSION TITLE
小白请问win端密码显示乱码怎么办？

### DISCUSSION THREAD

**Discussion by xuzhe25728** [AlistGo/alist#868] [Q&A] [answered]
不会其他安装方法，用的win端，在服务器添加端口规则后终于可以登录后台了，但是密码不对。别人都是只有几个字符，我的却一长串，如图
![2022-03-31_094603](https://user-images.githubusercontent.com/56471323/160961156-0e3e5a84-26a0-479c-9711-ba21921e502a.png)

**Comment by xhofe** [comment, marked as answer]
低版本Windows（win10以下）不适配，密码为4m和⬅之间的八个字符。

**Reply by xuzhe25728** [reply]
谢谢大佬
